.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Description
+
+(PR description goes here)
+
+### Checklist
+
+- [ ] `README.md` has been updated or is not required
+- [ ] push trigger tests
```

Comment:
push to `main`, `develop` or both?

Reply:
And also: we want that tests executed on that repo: [test-addonfactory-repo](https://github.com/splunk/test-addonfactory-repo) ?